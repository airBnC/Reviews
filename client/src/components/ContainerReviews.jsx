import React from 'react';

import ReviewEntry from './ReviewEntry';

const ContainerReviews = props => (
  <div>
    <h3> Reviews Container </h3>
    <ReviewEntry />
  </div>
);

export default ContainerReviews;
