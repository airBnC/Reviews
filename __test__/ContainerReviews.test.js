import React from 'react';
import { configure, shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import ContainerReviews from '../client/src/components/ContainerReviews';

configure({ adapter: new Adapter() });

describe('ContainerReviews', () => {
  it('should render correctly', () => {
    const output = shallow(<ContainerReviews />);
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});