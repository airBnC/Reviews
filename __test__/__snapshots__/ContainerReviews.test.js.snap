// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContainerReviews should render correctly 1`] = `
<div>
  <h3>
     Reviews Container 
  </h3>
  <ReviewEntry />
</div>
`;
